.rubocop.yml

```diff
@@ -791,6 +791,7 @@ Style/PredicateName:
   # helpers in the form of `have_something` or `be_something`.
   Exclude:
     - '**/spec/**/*'
+    - '**/test/**/*'
```

Comment:
I added some helpers (beginning with `have_`) to a file that lives in a folder named `test`. So I exclude those paths here. Hovewer, I think the better fix would be to actually rename the folders to `spec`?